### PR TITLE
feat(@meso-network): :zap: initial release of meso-js

### PR DESCRIPTION
**Pull request by kyledetella**
This sets up the `@meso-network/*` packages for publishing.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+20
```
Comment (dseeto) [inline .nvmrc:1]: What is this `20` here?

File: cspell.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "CHANGELOG.md",
+    "lib"
+  ],
+  "dictionaryDefinitions": [],
+  "dictionaries": [],
+  "words": [
+    "allowtransparency",
+    "caip",
+    "commitlint",
+    "delivr",
```
Comment (dseeto) [inline cspell.json:17]: Should this be here?

File: packages/post-message-bus/package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@meso-network/post-message-bus",
+  "version": "0.0.58",
```
Comment (dseeto) [inline packages/post-message-bus/package.json:3]: Did you want to reset this too?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "meso-js",
+  "version": "0.0.58",
```
Comment (dseeto) [inline package.json:3]: How about this?

File: packages/types/package.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@meso-network/types",
+  "version": "0.0.58",
```
Comment (dseeto) [inline packages/types/package.json:3]: And this?

**Review by dseeto — CHANGES_REQUESTED** [review]

File: .nvmrc (hunk 1 above)
Comment (kyledetella) [reply .nvmrc:1]: Version 20 of nodejs. For [nvm](https://github.com/nvm-sh/nvm)

File: cspell.json (hunk 2 above)
Comment (kyledetella) [reply cspell.json:17]: It was in the old docs. Can remove.

Comment (kyledetella) [reply cspell.json]: Fixed.

**Review by dseeto — APPROVED** [review]

**Pull request merged**